src/client/dfuse/dfuse_obj_da.c

```diff
@@ -26,6 +26,7 @@ struct da_entry {
 };
 
 struct obj_da {
+	int magic;                 /* magic number for sanity */
```

Comment:
What's this change for?  The only way I can see it making sense is if pthread_key_t is 32bit.

Reply:
the compile time size of the struct adds 4 byte pad for each before this change. putting them together would avoid that. the issue was a compile time assertion that was exposed on aarch64 because of the large mutex size there. 
for more info see slack discussion, you were tagged.

Reply:
alternatively we could have increased the size in obj_da.h to 136 instead, but the fix here looks better to me.

Reply:
Yes, I normally try and group smaller entries together, I must have just assumed pthread_key_t was 64 bit.